client/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java

```diff
@@ -0,0 +1,378 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.service.center.client;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Map;
+
+import javax.management.OperationsException;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.servicecomb.service.center.client.http.*;
+import org.apache.servicecomb.service.center.client.model.HeartbeatsRequest;
+import org.apache.servicecomb.service.center.client.model.Microservice;
+import org.apache.servicecomb.service.center.client.model.MicroserviceInstance;
+import org.apache.servicecomb.service.center.client.model.MicroserviceInstanceStatus;
+import org.apache.servicecomb.service.center.client.model.MicroserviceInstancesResponse;
+import org.apache.servicecomb.service.center.client.model.MicroservicesResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by   on 2019/10/16.
+ */
+public class ServiceCenterClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ServiceCenterClient.class);
+
+    private ServiceCenterRawClient httpClient;
+
+    /**
+     * Use default config parameter
+     */
+    public ServiceCenterClient() {
+        this(new ServiceCenterRawClient());
+    }
+
+    /**
+     * Add TLS config of client
+     * @param tlsConfig
+     */
+    public ServiceCenterClient(TLSConfig tlsConfig){
+        this(tlsConfig,null);
+    }
+
+    /**
+     * Add extraGlobalHeaders to http request
+     * @param extraGlobalHeaders
+     */
+    public ServiceCenterClient(Map<String, String> extraGlobalHeaders) {
+        this(null,0,null,null,extraGlobalHeaders);
+    }
+
+    /**
+     * Add TLS config and extraGlobalHeaders
+     * @param tlsConfig
+     * @param extraGlobalHeaders
+     */
+    public ServiceCenterClient(TLSConfig tlsConfig,Map<String, String> extraGlobalHeaders){
+        this(null,0,null,tlsConfig,extraGlobalHeaders);
+    }
+
+    /**
+     * Customized host,port,
+     * @param host
+     * @param port
+     */
+    public ServiceCenterClient(String host, int port) {
+        this(host, port, null,null, null);
+    }
+
+    /**
+     * Customized host,port,domainName,TLSConf, headers and any one parameter can be null.
+     * @param host
+     * @param port
+     * @param domainName
+     * @param tlsConfig
+     * @param extraGlobalHeaders
+     */
+    public ServiceCenterClient(String host, int port, String domainName, TLSConfig tlsConfig,  Map<String, String> extraGlobalHeaders) {

```

Comment:
This module has a lot of same code as kei-client, maybe a common http-client module can be created

Reply:
yes, this is a good idea.

Reply:
Can you fix the format problems and some other problems you want to fix first? And I can merge the two PR's first to help you do refactoring easier.

Reply:
ok , thanks. 
The code format is now standardized and some problem is fixed.